src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -227,6 +233,44 @@ const GroupCreationModal: React.FC<GroupCreationModalProps> = ({
           createNavigate(generateConversationUrl(conversation.qualifiedId))(event);
         }
       } catch (error) {
+        const NON_FEDERATING_BACKENDS = 409;
```

Comment:
Move it above component :)

Reply:
and maybe add the import for
```
import {StatusCodes as HTTP_STATUS} from 'http-status-codes';
```
and
```suggestion
        const NON_FEDERATING_BACKENDS = HTTP_STATUS.CONFLICT.;
```